SharedPackages/AttributedMetric/Tests/AttributedMetricTests/TimePastTests.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  TimePastTests.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+@testable import AttributedMetric
+
+final class TimePastTests: XCTestCase {
```

Comment:
This could be renamed now to match the name of the class it's testing

Reply:
renamed